### ISSUE TITLE
Not working properly

### ISSUE THREAD

**Issue by aman5898** [Hikari9/comfortable-swipe#66]
Got installed correctly but when check for status it says in the last line, comfortable-swipe program is stopped.
On comfortable-swipe debug it says failed to run command 'libinput' : no such file or directory.
![Screenshot from 2019-04-07 12-46-18](https://user-images.githubusercontent.com/32238811/55680146-54a7d480-5933-11e9-90d8-936dbc8678b7.png)

**Comment by Hikari9** [comment]
Have you installed `libinput-tools` from installation step 1?

```bash
sudo apt update && sudo apt install libinput-tools
```

Make sure to also follow **all steps** from "How to Run" section.

**Comment by aman5898** [comment]
yes, I have installed them still the same problem.
![Screenshot from 2019-04-07 18-51-08](https://user-images.githubusercontent.com/32238811/55684221-635baf00-5966-11e9-904c-798d77d8d453.png)

**Comment by Hikari9** [comment]
Thanks for pointing that out. This may indeed be a problem with comfortable swipe installation. Please try out the following as workarounds:

1. Does running `libinput-debug-events` work but not `libinput debug-events`?

    - If yes, try adding the extra dash `-` in `lib/service/debug.cpp` then run `bash install` again.

2. Try `sudo apt install libinput-dev` before running `comfortable-swipe start`again.

    - If this works, I'll add the missing install to the Readme.

Hope any one of the above solutions work for you. I'll add a PR to fix the bug depending on your input.

**Comment by aman5898** [comment]
Can you please explain what to do in the first part of instruction more clearly ???????

**Comment by Hikari9** [comment]
TLDR; try to check if this runs:

```
libinput-debug-events
```

## Long Explanation

As of Ubuntu 18.04, libinput said that they would **deprecate** the command `libinput-debug-events` (with the dash) in favor of `libinput debug-events` (without the dash).

This change was adapted recently for comfortable-swipe (see PR #31 and #32).

However, I fear this change might **NOT** be backwards-compatible with your 16.04 version, hence my first instruction.

Hope that clears it up. Please comment if it fixes the issue, then I will make a PR for the fix shortly.

**Comment by ad1tyak** [comment]
In case this is still unclear, I'm using 16.04 and using `libinput-debug-events` worked for me and it works perfectly now, Thank you.
